### ISSUE TITLE
Can't get it to work on Windows (7)

### ISSUE THREAD

**Issue by Octopixell** [fourseven/meteor-scss#94]
I can't seem to get this package to work on Windows. On my Mac it works like a charm and it's the perfect solution but on Windows it's not working for me. 

my `scss.json` file:
```
{
  "sourceMap": false,
  "enableAutoprefixer": true,
  "autoprefixerOptions": {
    "browsers": ["last 20 versions"],
    "cascade": false
  },
  "useIndex" : true,
  "indexFilePath" : "client/scss/main.scss",
  "includePaths": [
    ".meteor/local/bower"
  ]
}
```

the `main.scss` file made from the `indexFilePath` option:

```
// This file is auto generated by the scss package
// New .scss and .sass files will be automatically '@import'ed  at the bottom
// Existing content in the file will not be touched
// When deleting a .scss or .sass file you must manually delete it from here

@import '.meteor\local\bower\bourbon\app\assets\stylesheets\bourbon';

@import 'client\scss\partials\variables';

@import '.meteor\local\bower\neat\app\assets\stylesheets\neat';

@import 'client\scss\partials\mixins';

@import 'client\scss\partials\utilities';

@import "client\scss\style.scss";

@import "client\scss\account.scss";

@import "client\scss\main.scss";
```

First thing I saw was that the paths use `\` instead of `/` and I'm not sure if that's just Windows or a bug. As you can also see it keeps importing the `main.scss` file itself as well which doesn't happen on Mac.

Basically, it recognizes all the `.scss` files but it doesn't seem to compile anything and nothing gets changed on the front-end.

Any ideas what might be going wrong? I'm running the latest version (`v3.1.1`). Please let me know if you need more information :smile:

**Comment by Octopixell** [comment]
Note: I've manually added the `@import '.meteor\local\bower\bourbon\app\assets\stylesheets\bourbon';` and `
@import '.meteor\local\bower\neat\app\assets\stylesheets\neat';` because I need it in this specific load order. (Setting some neat variables in `@import 'client\scss\partials\variables';`

**Comment by iamlasse** [comment]
Still not fixed, just ran into same problem here with my team

**Comment by mariocatch** [comment]
This is still occurring. It's also including the index file at the bottom of itself, which causes no styles to be applied to the site when ran.

**Comment by DevanB** [comment]
I too am having issues getting it to work on Windows. Works perfectly on Mac though.

I'll be glad to share anything necessary to help fix the problem.

**Comment by Octopixell** [comment]
Hmm it's really too bad over a month has passed and we haven't even heard a thing.. I'm not sure how to resolve the issue myself, I'll try and have a look at the code to see if I can have a go at making a fix.. Haven't looked into Meteor Packages at all yet so I'm unsure, not making any promises ;)

**Comment by DevanB** [comment]
I don't typically work on Windows (haven't touched one in several months until today when I ran into the problem), but I still have to make sure I choose a tool that works with all the other devs that are on Windows.

I don't really even see another package to add Sass/SCSS support to Meteor...

**Comment by Octopixell** [comment]
I hate working on Windows to be honest, but that's all I got @ home.. This is the best solution for using Sass/SCSS in Meteor as far as I'm concerned.. Having to run a virtual PC just to use this package is doing my head in though ;) haha. I'll have a look soon.

**Comment by DevanB** [comment]
Until this is resolved, I'm just going to manually compile on my Mac into css.

Let me know if I can help any!

**Comment by fourseven** [comment]
Hi - so are you all experiencing problems because of useIndex? 

I do very little Windows development but I'm looking into this now.

**Comment by DevanB** [comment]
I was utilizing `useIndex` and `indexPath`, but I'm not sure if that is the root of the problem.

Thanks for looking into it! I'd love to have to stop manually compiling the Sass/SCSS

**Comment by rfox90** [comment]
I've taken to renaming the index file to index.xscss to prevent this.

kinda messy but it prevents the looping include

**Comment by fourseven** [comment]
Yeah it's because (in @Octopixell's case) client/scss/main.scss doesn't equal client\scss\main.scss - windows is returning and saving the paths with the reverse slashes, and it all goes horribly wrong. I'm just working out the best way to resolve the paths now, `String.prototype.replace()` or some fs/path method

**Comment by fourseven** [comment]
Can I get a second person to test these fixes please? You just need to clone down this repo into your `$PROJECT_ROOT/packages/` folder and restart the server, master should have the fix (please re-adjust any `\` in the index scss file to be `/`) and restart your server.

@rfox90 @DevanB @Octopixell @iamlasse @mariocatch

**Comment by DevanB** [comment]
I initially saw the `\main.scss` issue but I refreshed the file in WebStorm and those lines weren't there anymore.

I'll be glad to test it out. Booting up the ole Windows 7 now! @fourseven

**Comment by fourseven** [comment]
Sorry I didn't see the update @DevanB - you'll need to have fourseven:scss added to your project, and if it's in the packages sub folder it will use that over the released package.

It's /packages/meteor-scss in my version and reads it correctly

**Comment by DevanB** [comment]
Alright, I'll give it a shot.

**Comment by DevanB** [comment]
I'm happy to report back that 3.2.0 is successfully working correctly. I noticed that my `app.sass` is also not `@import` itself like referenced in #98. Also, the directory structure is not using back slashes like referenced in #100. 

This was tested on Windows 7 Enterprise Edition.

**Comment by fourseven** [comment]
3.2.0 is released, please let me know if there's still an issue. You may need to re-write the index.scss file you use to utilize `/` instead of `\`

**Comment by Octopixell** [comment]
@fourseven you're a hero :+1: Now I can continue my development :smile:

**Comment by rfox90** [comment]
Thanks a bunch!
